pandas/core/generic.py

```diff
@@ -3636,6 +3637,34 @@ def astype(self, dtype, copy=True, errors='raise', **kwargs):
         Returns
         -------
         casted : type of caller
+
+        Examples
```

Comment:
Let's provide an example using the `copy` argument given that it says the parameter should be handled with care.